docs/core/deploying/trimming/snippets/ConsoleApp1/ConsoleApp1.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <TargetFramework>net8.0</TargetFramework>
+    <PublishTrimmed>true</PublishTrimmed>
+    <PublishAot>true</PublishAot>
```

Comment:
Since this topic is about trimming - would leave out `PublishAot=true` - it does more than just trimming and thus would make it confusing, since people would start getting warnings unrelated to trimming (`IL3050` for example).